config/custom_components/ninjasphere.py

```diff
@@ -0,0 +1,67 @@
+import homeassistant.components.light as light
+import homeassistant.components.media_player as media_player
+from .ninjasphere_components.light import Light
+from .ninjasphere_components.mediaplayer import MediaPlayer
+from .ninjasphere_components.unknownthing import UnknownThing
+
+import pyninjasphere.services.service as service
```

Comment:
De imports mogen niet hier. Zie de docu op

Home assistent gaat een  pip install doen op basis van:
REQUIREMENTS = ['pyninjasphere==0.0.1.dev0']

En in de setup mag je vervolgens pas de imports doen:

    from pyninjasphere.services import service